### ISSUE TITLE
[cscd-llh] This project is not importable in python (Trac #1655)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1647]
This is a pretty easy fix.  Give it a python dir, point cmake to it, add an __init__.py that loads the library.

Migrated from https://code.icecube.wisc.edu/ticket/1655
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:14:55", 
    "description": "This is a pretty easy fix.  Give it a python dir, point cmake to it, add an __init__.py that loads the library.", 
    "reporter": "olivas", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067295757382", 
    "component": "combo reconstruction", 
    "summary": "[cscd-llh] This project is not importable in python", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2016-04-25T19:01:34", 
    "milestone": "", 
    "owner": "tpalczewski", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160425T19:04:45: david.schultz commented:

> If a library doesn't have pybindings, does it make sense to import it from python? Does it even work properly?

**Comment by icecube-trac** [comment]
Trac update at 20160425T19:07:11: olivas commented:

> It does and it works just fine.  All we're doing is moving the load behind the scenes to allow for a common syntax.  User's shouldn't have to remember ('cause I sure don't) which projects do and do not have pybindings.

**Comment by icecube-trac** [comment]
Trac update at 20160426T17:12:15: 
* olivas commented:

> In changeset 145078:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="145078"
> this is now importable in python. closes #1647.
> ```


* olivas changed status from "new" to "closed"
* olivas changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:14:55: 
* olivas changed milestone from "IceRec 5.2" to ""
* olivas commented:

> Milestone IceRec 5.2 deleted
